packages/front-end/components/Settings/Team/MemberList.tsx

```diff
@@ -34,7 +34,7 @@ const MemberList: FC<{
 
   return (
     <div className="my-4">
-      <h5>Active Members</h5>
+      <h5>Active Members{` (${Array.from(users.values()).length})`}</h5>
```

Comment:
Map objects have a `size` property you can use.  So this can just become `users.size`.